src/http/http_endpoint.h

```diff
@@ -51,26 +51,54 @@ namespace enclave
 
       if (!is_websocket)
       {
+        auto buf = read(4096, false);
+        auto data = buf.data();
+        auto size = buf.size();
+
         while (true)
         {
-          auto buf = read(4096, false);
-          if (buf.size() == 0)
+          if (size == 0)
           {
             return;
           }
 
           LOG_TRACE_FMT(
             "Going to parse {} bytes: \n[{}]",
-            buf.size(),
-            std::string(buf.begin(), buf.end()));
+            size,
+            std::string((char const*)data, size));
```

Comment:
We shouldn't keep this around - trying to print the binary payload of a HTTP request will cause us pain. But its a very useful debugging line for now.